### DISCUSSION TITLE
ZeppOS development, getting Barometer information

### DISCUSSION THREAD

**Discussion by meirtolpin11** [zepp-health/zeppos-forum#65] [Q&A] [answered]
Hello, 
I want to develop a "pilot friendly" barometer app to Amazfit gtr 4. 
For that I need to be able to fetch current air pressure from the barometer sensor. 
Is it possible with current SDK?
And where can I find the SDK documentation?

Thank you!

**Comment by zdunex25** [comment, marked as answer]
You have to wait for ZeppOS 3.0 as for now it is not supported.
https://docs-testing.zepp.com/docs/next/reference/device-app-api/newAPI/sensor/Barometer/

**Reply by meirtolpin11** [reply]
There is any estimate for the release date?

**Reply by zdunex25** [reply]
It should come along with new series of watches as usually. Documentation is officially online so should not take very long although there is no ETA.

**Reply by meirtolpin11** [reply]
And is there a chance that GTR 4 will support ZeppOs 3.0?

**Reply by zdunex25** [reply]
Hard to tell honestly. Amazfit usually releases new os with new device, there is no a single zeppos 1.0 device that has been updated to v2, but if you follow the link I shared you will notice barometer is supported since v2.1. There is a chance gtr 4 will get 2.1 update.

**Comment by joaquinperaza** [comment]
Hi, it is now supported in API 2.1 (I am waiting for testing on GTR4 currently incompatible with 2.1) I have the first version coded, would you like to work together? I plan to build the rest of aviation gauges and maybe some minimal map with nearest airport (kind of Poor man garmin) 😂

**Reply by meirtolpin11** [reply]
Sounds great, Do you know if GTR 4 will be ever compatible with 2.1 ? 
I will be happy to work together, lets chat about this :) 
my email is - meirtolpin11@gmail.com

**Reply by zdunex25** [reply]
They confirmed it will, no ETA tho.
